flake.nix

```diff
@@ -8,19 +8,42 @@
   };
 
   outputs = inputs@{ self, nixpkgs, flake-utils, rust-overlay, ... }:
-    flake-utils.lib.eachDefaultSystem (system: 
+    flake-utils.lib.eachDefaultSystem (system:
       let
-        pkgs = import nixpkgs { inherit system; overlays = [ rust-overlay.overlays.default ]; };        
+        pkgs = import nixpkgs { inherit system; overlays = [ rust-overlay.overlays.default ]; };
         devnet-hardhat = import ./ops/scripts { inherit pkgs; };
```

Comment:
Overall looks good!

We can also remove `default.nix` in `./ops/scripts` and the line below then keep the derivations in this file to simplify it further

```suggestion
```